src/Common/PasswordManipulation.php

```diff
@@ -35,7 +35,7 @@ class PasswordManipulation
      *
      * @since 3.0.0
      *
-     * @return int
+     * @return mixed
```

Comment:
Why password length is `integer`?

Could you explain that?

Reply:
its return value, it return mixed, int or false